python/GafferCortexUI/OpDialogue.py

```diff
@@ -120,6 +122,8 @@ def __init__(
 
 		GafferUI.Dialogue.__init__( self, title, sizeMode=sizeMode, **kw )
 
+		self._qtWidget().installEventFilter( _opDialogueEventFilter )
```

Comment:
Do you know why this is necessary? I suspect it is because `GafferCortexUI.CompoundPlugValueWidget` doesn't build its UI until it becomes visible. I wonder if a simpler solution might be to just have it build the UI in `CompoundPlugValueWidget.__init__`?